schema-admin.graphql

```diff
@@ -345,6 +345,10 @@ type ScheduledCuratedCorpusItem {
     The associated Approved Item.
     """
     approvedItem: ApprovedCuratedCorpusItem!
+    """
+    Optional, The GUID of this schedudedSurface to which this item is scheduled. Example: 'NEW_TAB_EN_US'.
+    """
+    scheduledSurfaceGuid: ID
```

Comment:
We'll never have a scheduled item without a surface it was scheduled on, so I think it's safe to make this prop required